pkg/registry/networking/ingress/strategy.go

```diff
@@ -67,11 +69,14 @@ func (ingressStrategy) PrepareForUpdate(ctx context.Context, obj, old runtime.Ob
 
```

Comment:
If an update of an ingress with a spec.class value drops the spec.class value in the new incoming object (which a client using an old version of client-go would do), should we preserve the existing value in `PrepareForUpdate`? alternately, should the ingressclass admission plugin also populate the default class on update for ingresses that don't have it or the class annotation set?